rts/lra/lra-test/tck/pom.xml

```diff
@@ -25,6 +25,7 @@
         <lra.tck.suite.thorntail.trace.params></lra.tck.suite.thorntail.trace.params>
         <lra.tck.suite.debug.params></lra.tck.suite.debug.params> <!-- has content when -Ddebug.tck is specified -->
         <lra.tck.suite.debug.port>8788</lra.tck.suite.debug.port>
+        <lra.tck.consistency.delay>2000</lra.tck.consistency.delay>
```

Comment:
Side comment: Would it make sense to make a timeouts in spec a little bigger? IIRC this will apply 2s to every complete / compensate etc. right? It doesn't hurt of course, but it is possible that timelimit test numbers are just low.

Reply:
I'm not ready yet - I will clean up the PR when I've seen the TCK passing